app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V2::ActivationKeysController < Api::V2::ApiController
 
     before_action :verify_presence_of_organization_or_environment, :only => [:index]
     before_action :find_environment, :only => [:index, :create, :update]
-    before_action :find_optional_organization, :only => [:index, :create, :show]
+    before_action :find_optional_nested_object, :only => [:index, :create, :show]
```

Comment:
As for nested_resource, it's the first resource when you have 2 in the API. e.g:

`katello/api/v2/orgnaizations/:organization_id/products/:id` 

Product is the resource, Organization is the nested resource. Certainly if it requires an explanation it means the name is not very readable...